library/src/enabled/java/com/telefonica/tweaks/TweaksTheme.kt

```diff
@@ -49,7 +49,7 @@ fun DefaultTweaksTheme(
         tweaksBackground = TweaksDarkBlue,
         tweaksOnBackground = Color.White,
         tweaksGroupBackground = TweaksDarkBlueBackground,
-        tweaksColorModified = TweaksDarkBlueBackground,
+        tweaksColorModified = TweaksGreen,
```

Comment:
GroupBackground color and modified label color was the same. So it couldn't be seen.